server.js

```diff
@@ -20,8 +20,32 @@ app.get('/', (req, res) => {
     res.send(`<h1>This server is running on PORT ${PORT}</h1>`);
 });
 
-// ==== Helper functions ====
+app.get('/location', (req, res) => {
```

Comment:
Applicable to the WRRC diagram this is the red request.
- line 23 is red
Request is on:
- line 39 is green